src/entrypoints/my-redirect.ts

```diff
@@ -56,6 +62,12 @@ const render = (showTroubleshooting: boolean) => {
     return;
   }
 
+  if (!isMobile && navigator.userAgent.includes("Windows")) {
```

Comment:
I do like this idea, but agree with Bram that the user should never aware of this. 

Just allowing it for Windows is ok. But prefer if we could also allow it for Chromium browsers on Mac, but that can be in a subsequent PR.

I will ask a Windows users to verify the behavior and then this can be merged. Definitely better than another button.